includes/Story_Post_Type.php

```diff
@@ -473,14 +473,16 @@ public function admin_enqueue_scripts( $hook ) {
 			$script_dependencies[] = 'postbox';
 		}
 
-		$this->enqueue_script( self::WEB_STORIES_SCRIPT_HANDLE, $script_dependencies );
+		$all_scripts = $this->enqueue_script( self::WEB_STORIES_SCRIPT_HANDLE, $script_dependencies );
 		$this->enqueue_style( self::WEB_STORIES_SCRIPT_HANDLE, [ 'roboto' ] );
 
-		wp_localize_script(
-			self::WEB_STORIES_SCRIPT_HANDLE,
-			'webStoriesEditorSettings',
-			$this->get_editor_settings()
-		);
+		foreach ( $all_scripts as $script ) {
+			wp_localize_script(
+				$script,
+				'webStoriesEditorSettings',
+				$this->get_editor_settings()
+			);
+		}
```

Comment:
Same here. Only **the main script** needs to be localized. `$all_scripts` is not needed.